### ISSUE TITLE
2.4.1: [armhf] test failure: TestRestoreOld (unreliable test?)

### ISSUE THREAD

**Issue by onlyjob** [docker/distribution#1927]
On _armhf_ **TestRestoreOld** fails as follows:

```
-- FAIL: TestRestoreOld (0.05s)
    scheduler_test.go:85: ref1 should be removed first
```

See [full build log](https://buildd.debian.org/status/fetch.php?pkg=docker-registry&arch=armhf&ver=2.4.1~ds1-2&stamp=1471497818).

**Comment by RichardScothern** [comment]
Tests should be fixed by #1925

**Comment by stevvooe** [comment]
@onlyjob Let us know if you're still having this problem.
